web-portal/components/admin-venue-edit/AdminEventsListRow.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <tr>
+    <td>
+      <img
+        v-if="calendar_event.reviewed_by_org"
+        :src="ownerLogo"
+        width="30"
+      />
+
+    </td>
+
+    <td>{{ calendar_event.title }}</td>
+
+    <td>
+      <div v-for="(date_time, index) in calendar_event.date_times.slice(0, 2)" :key="index">
+        {{ dateFormat(date_time) }}
+      </div>
+
+      <small v-if="calendar_event.date_times.length > 2">
+        (and {{ calendar_event.date_times.length - 2 }} more)
+      </small>
```

Comment:
If we wanted to handle this with a computed property, we could have it return an object with one or two pre-formatted dates + the count of how many more there are, something like:

```js
when: function () {
  return {
    formattedDates: calendar_event.date_times?.slice(0, 2).map((dt) => this.dateFormat(dt)),
    more: calendar_event.date_times?.length > 2 ? calendar_event.date_times.length - 2 : 0
  }
}
```

And then the template could use that like this:

```vue
      <div v-for="(formattedDate, index) in when.formattedDates" :key="index">
        {{ formattedDate }}
      </div>

      <small v-if="when.more">
        (and {{ when.more }} more)
      </small>
```

Reply:
thx, probably just leave it alone for now